src/main/java/bio/terra/service/load/LoadDao.java

```diff
@@ -196,6 +199,26 @@ public int getBatchSize() {
         });
   }
 
+  /**
+   * Chunk a stream of models into groups of 1000 and populate the database
+   *
+   * @param loadId Load ID tying all these file ingests together
+   * @param loadFileModelStream The stream to be chunked and processed over
+   */
+  public void populateFiles(
+      UUID loadId, Stream<BulkLoadFileModel> loadFileModelStream, int batchSize) {
+    Spliterator<BulkLoadFileModel> split = loadFileModelStream.spliterator();
+
+    while (true) {
+      List<BulkLoadFileModel> batch = new ArrayList<>(batchSize);
+      for (int i = 0; i < batchSize; i++) {
+        split.tryAdvance(batch::add);
+      }
+      if (batch.isEmpty()) break;
```

Comment:
all blocks should have `{}`s
```suggestion
      if (batch.isEmpty()) {
        break;
      }
```